modules/ROOT/pages/index-self-managed.adoc

```diff
@@ -75,15 +128,20 @@ Follow best practices by installing Runtime Fabric on Self-Managed Kubernetes in
 * At least 15 GiB of RAM
 * At least 250 GiB of available disk space
 
-Adjust the number of nodes and amout of resources allocated according to the amount and type of workload you run on each Runtime Fabric.
+Adjust the number of nodes and amount of resources allocated according to the amount and type of workload you run on each Runtime Fabric.
 
-=== Supported Architectures
+=== Anypoint Platform Roles and Permissions
 
-Runtime Fabric on Self-Managed Kubernetes requires worker nodes which use the x86/x64 architecture. ARM-base architectures are not supported.
+To succesfully use Runtime Fabric, your Anypoint Platform account must have the following permissions enabled:
 
-=== Operating Systems
+* To manage permissions for Anypoint Platform users, you must have the ability to use xref:access-management::index.adoc[Anypoint Access Management].
+* To deploy and manage applications, you must have the ability to use Anypoint Runtime Manager. To deploy applications, you must also have the Exchange Contributors permission enabled for your Anypoint Platform account.
+* To use Runtime Fabric, you must have the Organization Administrators permission or the Manage Runtime Fabrics permission on the corresponding environments.
+* To delete Runtime Fabric instances, administrators need the Manage Runtime Fabrics permission at the organization level.
 
-Runtime Fabric on Self-Managed Kubernetes supports any Linux-based operating system supported by Amazon EKS, AKS, or GKE.
+=== Network Configuration
+
+Runtime Fabric on Self-Managed Kubernetes requires an IT administrator to configure network ports, hostnames, and certificates to function correctly. See xref:install-self-managed.adoc[Install Runtime Fabric on Self-Managed Kubernetes].
```

Comment:
```suggestion
Anypoint Runtime Fabric on Self-Managed Kubernetes requires an IT administrator to configure network ports, hostnames, and certificates to function correctly. See xref:install-self-managed.adoc[Install Runtime Fabric on Self-Managed Kubernetes].
```

Reply:
I have changed the naming in text in which I had other edit comments, but I did not change all naming. Please search globally and make precise each instance of Runtime Fabric (to either Anypoint Runtime Fabric or to Anypoint Runtime Fabric on Self-Managed Kubernetes).
If none of the instances refers to "vanilla" Anypoint Runtime Fabric, perhaps the distinction does not matter if we make that perfectly clear.
As I said, I am checking with MarComm on official abbreviations for the self-managed versions for EKS, AKS, and GKS.

Reply:
CONFIRMED